nemo/utils/precision_utils.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import torch
+
+
+def is_bfloat16_available():
```

Comment:
Why not to use `torch.cuda.is_bf16_supported()`?

Reply:
Because only the most recent pytorch supports that

Reply:
Oh, I see, thanks!

Reply:
Do we support pytorch 1.9 and earlier? It was added in [1.10.0](https://github.com/pytorch/pytorch/releases/tag/v1.10.0)

Reply:
We don't, min is 1.11. Guess we can use that then. Dima is looking into a more proper check anyway